core/common/src/ConcurrentQuery.ts

```diff
@@ -129,64 +134,143 @@ export interface BlobOptions extends BaseReaderOptions {
     return this;
   }
+  /**
+   * By default all blobs are abbreviated to save memory and network bandwidth. If set to false, all blob data will be returned by query as is.
+   * Use @type BlobReader to access blob data more efficiently.
+   * @param val A boolean value, if set to false will return complete blob type property data. This could cost time and network bandwidth.
+   * @returns @type QueryOptionsBuilder for fluent interface.
+   */
   public setAbbreviateBlobs(val: boolean) {
     this._options.abbreviateBlobs = val;
     return this;
   }
+  /**
+   * When query fail to prepare it will log error. This setting will suppress log errors in case where query come from user typing it and its expected to fail often.
+   * @param val A boolean value, if set to true, any error logging will be suppressed.
+   * @returns @type QueryOptionsBuilder for fluent interface.
+   */
   public setSuppressLogErrors(val: boolean) {
     this._options.suppressLogErrors = val;
     return this;
   }
+  /**
+   * If set ECClassId, SourceECClassId and TargetECClassId system properties will return qualified name of class instead of a @typedef Id64String.
+   * @param val A boolean value.
+   * @returns @type QueryOptionsBuilder for fluent interface.
+   */
   public setConvertClassIdsToNames(val: boolean) {
     this._options.convertClassIdsToClassNames = val;
     return this;
   }
+  /**
+   * Specify limit for query. Limit determine number of rows and offset in result-set.
+   * @param val Specify count and offset from within the result-set of a ECSQL query.
+   * @returns @type QueryOptionsBuilder for fluent interface.
+   */
   public setLimit(val: QueryLimit) {
     this._options.limit = val;
     return this;
   }
+  /**
+   * Specify row format returned by concurrent query manager.
+   * @param val @enum QueryRowFormat specifying format for result.
+   * @returns @type QueryOptionsBuilder for fluent interface.
+   */
   public setRowFormat(val: QueryRowFormat) {
     this._options.rowFormat = val;
     return this;
   }
-
+  /**
+   * Defers execution of query in queue by specified milliseconds. This parameter is ignored by default unless concurrent query is configure to not ignore it.
+   * @param val Number of milliseconds.
+   * @returns @type QueryOptionsBuilder for fluent interface.
+   */
+  public setDelay(val: number) {
+    this._options.delay = val;
+    return this;
+  }
 }
 /** @beta */
 export class BlobOptionsBuilder {
   public constructor(private _options: BlobOptions = {}) { }
   public getOptions(): BlobOptions { return this._options; }
+  /**
+   * Allow to set priority of blob request. Blob request will be inserted int queue base on priority value. This value will be ignored if concurrent query is configured with ignored priority is true.
+   * @param val integer value which can be negative as well. By default its zero.
+   * @returns @type BlobOptionsBuilder for fluent interface.
+   */
   public setPriority(val: number) {
```

Comment:
Does `priority` have any use outside of tests? If not, `@internal`. I don't think there's any public API for configuring concurrent query not to ignore either of those properties.